Sources/Sentry/Profiling/SentryProfilerSerialization.mm

```diff
@@ -110,6 +110,8 @@
         return @"backgrounded";
     case SentryProfilerTruncationReasonTimeout:
         return @"timeout";
+    case SentryProfilerTruncationReasonPanic:
+        return @"panic";
```

Comment:
`m`: It would be great to have a test for this.

Reply:
It's actually not needed. Removed.